helm-go-lib/go.mod

```diff
@@ -1,32 +1,15 @@
-module helm.sh/helm/v3@v3.2.4
+module broadinstitute/helm-scala-sdk
 
-go 1.14
+go 1.16
 
 replace (
-	// github.com/Azure/go-autorest/autorest has different versions for the Go
-	// modules than it does for releases on the repository. Note the correct
-	// version when updating.
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible
-	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
-
-	// Kubernetes imports github.com/miekg/dns at a newer version but it is used
-	// by a package Helm does not need. Go modules resolves all packages rather
-	// than just those in use (like Glide and dep do). This sets the version
-	// to the one oras needs. If oras is updated the version should be updated
-	// as well.
-	github.com/miekg/dns => github.com/miekg/dns v0.0.0-20181005163659-0d29b283ac0f
-	gopkg.in/inf.v0 v0.9.1 => github.com/go-inf/inf v0.9.1
-	gopkg.in/square/go-jose.v2 v2.3.0 => github.com/square/go-jose v2.3.0+incompatible
-
-	rsc.io/letsencrypt => github.com/dmcgowan/letsencrypt v0.0.0-20160928181947-1847a81d2087
+	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
+	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
 )
 
 require (
-	9fans.net/go v0.0.2 // indirect
-	github.com/Masterminds/semver v1.5.0 // indirect
-	github.com/helm/helm v2.16.6+incompatible // indirect
-	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
-	helm.sh/helm/v3 v3.2.0 // indirect
-	k8s.io/helm v2.16.6+incompatible // indirect
-	sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	helm.sh/helm/v3 v3.5.3
```

Comment:
Bumped `helm` and `k8s` to the latest. Also removed some dependencies which aren't needed.